templates/collectd.conf.j2

```diff
@@ -9,6 +9,13 @@ AutoLoadPlugin false
 
 TypesDB "/usr/share/collectd/types.db"{% if "libpodstats" in collectd_plugins %} "/usr/share/collectd/types.db.libpodstats"{% endif %}
 
+{% if collectd_write_queue_limit_high is defined %}
+WriteQueueLimitHigh {{ collectd_write_queue_limit_high }}
+{% endif %}
+{% if collectd_write_queue_limit is defined %}
```

Comment:
:-1: Should be `if collectd_write_queue_limit_low` I think?

Reply:
Indeed it should